pkg/pillar/cmd/zedmanager/handledomainmgr.go

```diff
@@ -77,6 +79,12 @@ func MaybeAddDomainConfig(ctx *zedmanagerContext,
 		disk.Target = vrs.Target
 		disk.CustomMeta = vrs.CustomMeta
 		dc.DiskConfigList = append(dc.DiskConfigList, disk)
+		// For NOHYPER type virtualization mode pass the KubeImageName to domainmgr
```

Comment:
What about the case of nohyper with kvm or xen?

Reply:
I updated the commit after Avi's comments. Removed the check for kubevirt eve. We now set the kubeImageName for all flavors of eve.

Reply:
I don't understand your response. 
This particular code seems to assume that if VirtualizationMode is NOHYPER then it should use ReferenceName. The comments are differently misleading since they talk about kubevirt eve, but the code applies even if kvm or xen.

Reply:
Yes the code applies to all flavors of EVE. But only on kubevirt eve pods will be launched using the KubeImageName. AFAIK, other flavors of eve do not use that KubeImageName to launch containers/pods. That's why I mentioned it explicitly. If that is confusing I can remove those comments

Reply:
I misread the comment.
But it wouldn't hurt to add punctuation and add one line to get
		// For NOHYPER type virtualization mode pass the KubeImageName to domainmgr. 
		// Pods will be launched using that KubeImageName in kubevirt eve.
// ReferenceName might be empty if not kuevirt eve, but KubeImageName will be ignored in that case anyhow.

Reply:
Will update the comments.